pywinauto/findwindows.py

```diff
@@ -200,10 +201,11 @@ def find_elements(class_name=None,
         # if not given a parent look for all children of the desktop
         if not parent:
             parent = backend_obj.element_info_class()
+        elif isinstance(parent, BaseWrapper):
+            parent = parent.element_info

```

Comment:
I'm not sure I understand it. Is the check for the instance of ```backend_obj.generic_wrapper_class``` at line 180 not enough ?